web/packages/shared/components/UnifiedResources/CardsView/ResourceCard.tsx

```diff
@@ -250,17 +223,17 @@ export function ResourceCard({
               <ResTypeIconBox>
                 <SecondaryIcon size={18} />
               </ResTypeIconBox>
-              {description.primary && (
-                <SingleLineBox ml={1} title={description.primary}>
+              {type && (
+                <SingleLineBox ml={1} title={type}>
                   <Text typography="body2" color="text.slightlyMuted">
-                    {description.primary}
+                    {type}
                   </Text>
                 </SingleLineBox>
               )}
-              {description.secondary && (
-                <SingleLineBox ml={2} title={description.secondary}>
+              {addr && (
+                <SingleLineBox ml={2} title={addr}>
                   <Text typography="body2" color="text.muted">
-                    {description.secondary}
+                    {addr}
```

Comment:
In a list view we show the description, but here we don't. Is this intentional? 
An idea that comes to my mind is to show one or another (`description || addr`, but I don't know which one is actually more important). I'd confirm it with design.

Reply:
This is intentional and I discussed it with design, the card layout doesn't allow for adding a description. From a UX perspective I'm not sure it's a good idea to have some cards show a description and others show an address in the same place as it could be confusing, I'll ask to see what Kenny thinks.

Reply:
This is intentional, it's because a description doesn't work well with the card's limited layout and space. I will keep the card layout the way it currently is in `master`.